src/app-layout/__tests__/runtime-drawers.test.tsx

```diff
@@ -31,7 +31,7 @@ jest.mock('@cloudscape-design/component-toolkit', () => ({
 }));
 
 async function renderComponent(jsx: React.ReactElement) {
-  const { container, rerender, getByTestId } = render(jsx);
+  const { container, rerender, getByTestId, ...rest } = render(jsx);
```

Comment:
I add such a statement every time to expose `debug` so I decided to leave it here